build/helm-operator/Dockerfile

```diff
@@ -1,35 +1,40 @@
 FROM golang:1.13-alpine as build
 
 ARG VERSION=1.0.0-rc9
-ARG KUBECTL_VERSION=v1.15.7
-ARG HELM_VERSION=v2.16.1
-ARG HELM3_VERSION=v3.1.1
 ARG TARGETPLATFORM
 
 ENV GO111MODULE=on \
     CGO_ENABLED=0
 
-RUN apk add --no-cache ca-certificates 'git>=2.12.0' curl
-
-ADD https://raw.githubusercontent.com/fluxcd/helm-operator/${VERSION}/docker/known_hosts.sh /known_hosts.sh
-ADD https://raw.githubusercontent.com/fluxcd/helm-operator/${VERSION}/docker/ssh_config /ssh_config
-ADD https://raw.githubusercontent.com/fluxcd/helm-operator/${VERSION}/docker/helm-repositories.yaml /helm-repositories.yaml
+RUN apk add --no-cache ca-certificates 'git>=2.12.0' curl bash
 
 WORKDIR /go/src/github.com/fluxcd/helm-operator
 
+SHELL ["/bin/bash", "-c"]
+
 RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
     export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2) && \
     GOARM=$(echo ${TARGETPLATFORM} | cut -d / -f3); export GOARM=${GOARM:1} && \
+    echo "GOOS=${GOOS}" && \
```

Comment:
Now we've tested this image we won't need the multiple `echo` commands to see what Go variables are set.

Reply:
I don't think having the debug output is bad here, in the case of docker buildx changes it might be nice to keep